spec/integration/aws_route53_hosted_zone_spec.rb

```diff
@@ -0,0 +1,375 @@
+require 'spec_helper'
+
+describe Chef::Resource::AwsRoute53HostedZone do
+  extend AWSSupport
+
+  when_the_chef_12_server "exists", organization: 'foo', server_scope: :context do
+    with_aws "when connected to AWS" do
+
+      # even though these resources are demonstrably convergent, `be_idempotent` fails, including when
+      # `update_aws_object` is nothing but an empty `converge_by` block. suggestions welcome.
```

Comment:
That'd be rad. Let me implement polling first (though I could use pointers from you on that as well), since some bugs are being hidden by Route 53's willingness to have multiple zones with the same name.